src/components/ButtonGroup/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "@typescript-eslint/no-unnecessary-type-arguments": "off"
```

Comment:
This rule is annoying and I think we should remove it. But didn't want to get to distracted